### ISSUE TITLE
w0w!

### ISSUE THREAD

**Issue by jonl1096** [nkrishn9/Computational-Genomics-Final-Project#1]
neural net prior deep learning perceptron posterior likelihood gibbs sampling expectation maximization gradient descent distribution non-linear optimization markov chain turing machine graphical model

**Comment by jonl1096** [comment]
Andrew Ng

**Comment by jonl1096** [comment]
@nkrishn9
